### PR TITLE
Add support for blackboard TX2 carrier

### PR DESCRIPTION
**Pull request by sjdrc**

### CODE REVIEW

File: blackboard-tx2.coffee [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+]
+
+module.exports =
+	version: 1
+	slug: 'blackboard-tx2'
+	aliases: [ 'blackboard-tx2' ]
+	name: 'Nvidia blackboard TX2'
+	arch: 'aarch64'
+	state: 'released'
+
```
Comment (floion) [inline blackboard-tx2.coffee]: Please add a community flag like here: https://github.com/balena-os/balena-jetson-tx2/blob/master/n510-tx2.coffee#L19

**Comment by floion** [comment]
@resin-jenkins ok to test

**Comment by floion** [comment]
@sjdrc also please follow the contribution guidelines: https://github.com/balena-os/balena-jetson-tx2#contributing

**Comment by floion** [comment]
Hi @sjdrc thanks for adding the Changelog-entry.
I would like for you to restructure this PR though. Can you split each change in it's own commit? So one commit per file? You could use this pull request as an example: https://github.com/balena-os/balena-jetson-tx2/pull/130

**Comment by sjdrc** [comment]
Sure, I was basing my work off https://github.com/balena-os/balena-jetson-tx2/pull/96

**Comment by sjdrc** [comment]
There, hopefully that is all good now.

**Comment by floion** [comment]
@balena-ci retest

**Comment by sjdrc** [comment]
Hey, now that this is merged, when can I expect to be able to provision this device from balenaCloud?

**Comment by floion** [comment]
It will be available in the following days.
Until then, seems we have forgotten about an icon file for it. Can you also push another PR with an icon file for your board please?

**Comment by sjdrc** [comment]
https://github.com/balena-os/balena-jetson-tx2/pull/140

**Comment by sjdrc** [comment]
It would be really great if it was available for provisioning before this weekend. We urgently need to test balena with our units before a deployment at the end of next week.

**Comment by floion** [comment]
We can have it deployed to the staging environment and you could test it from there. How does that sound?

**Comment by sjdrc** [comment]
Yep that would be great.

**Comment by sjdrc** [comment]
Hi, do you know when it will be available in staging?

**Comment by floion** [comment]
Hi, it is available in staging now

**Pull request merged**